libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -76,6 +117,29 @@ pair<string, string> IRGenerator::run(
 	return {warning + ir, warning + asmStack.print()};
 }
 
+void IRGenerator::verifyCallGraph(FunctionCallGraphBuilder::ContractCallGraph const& _graph)
+{
+	set<ASTNode const*, ASTNode::CompareByID> functions;
+
+	auto collectFromNode = [&](FunctionCallGraphBuilder::SpecialNode _node)
+	{
+		auto callees = _graph.edges.find(_node);
+
+		if (callees != _graph.edges.end())
+			for (auto callee: callees->second)
+				collectCalls(_graph, callee, functions);
+	};
+
+	collectFromNode(FunctionCallGraphBuilder::SpecialNode::CreationRoot);
+	collectFromNode(FunctionCallGraphBuilder::SpecialNode::CreationDispatch);
```

Comment:
I don't think this is needed if the graph is properly connected.

Reply:
Well, seem to have different understanding of "properly" ;) currently those two nodes are not connected at all.. should they be?